blocks/code/code.css

```diff
@@ -0,0 +1,350 @@
+/* NOTE: this is a modified version of the css from prism.js
+/* PrismJS 1.29.0
```

Comment:
Please let me know once the importer script is available as well?

Reply:
The import script is ready @nkthakur48